agents/components/Profile.js

```diff
@@ -105,11 +105,6 @@ export default compose(
     toggleEdit: ({ setEditing }) => () => setEditing(not)
   }),
   connectForm({
-    form: 'profile',
-    initialValues: {
```

Comment:
@ahdinosaur How can I fill the form with data from props? When I remove the initialValues the form is empty. I'm needing this to fill the Profile story with fake data.